src/shared/setup.js

```diff
@@ -136,7 +140,7 @@ const initialize = async function ({service, port, modules = [], runHandlers = f
       Logger.error(`No valid service type ${service} found!`)
       throw new Error(`No valid service type ${service} found!`)
   }
-
+  await Cache.initializeCache()
```

Comment:
I think it may only make sense to initialize this for the notificationHandler. So you should probably move it under the section that either starts ALL the handlers or just starts the notificationHandler. Let me know what you think about this.